Managers/FrameMan.cpp

```diff
@@ -26,12 +26,34 @@ namespace RTE {
 
 /////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
-	void FrameMan::DisplaySwitchOut(void) { g_UInputMan.DisableMouseMoving(true); }
+	void FrameMan::DisplaySwitchOut(void) { g_UInputMan.DisableMouseMoving(true);
+		#ifdef __unix__
+		// In Fullscreen regrab focus because the window is lost otherwise
+		FullscreenGrabFocus();
+		#endif
+	}
 
 /////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
 	void FrameMan::DisplaySwitchIn(void) { g_UInputMan.DisableMouseMoving(false); }
 
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+#ifdef __unix__
+	void FrameMan::FullscreenGrabFocus(){
```

Comment:
Just merge both of these together with the comments into `SetDisplaySwitchMode()`.
All the methods that reinit the game window call it so you don't need to call it from each one separately.

Reply:
grab focus needs to stay here.